src/backend/central/central_routes.py

```diff
@@ -20,10 +20,12 @@
 from sqlalchemy.orm import Session
 import logging.config
 import json
+import epdb
```

Comment:
Huh. Never come across this extended debugger before. No problem, but curious what you're using this for? It does add a dependency...

Reply:
I use it all the time, as it lets me set a breakpoint in the backend, which you can't do from the command line. This will go away at some point, but right now there's a lot of debug left to do. Epdb is better than the default pdb. Kristen uses ipdb, epdb is just what I'm used to. It's a stable dependency, but not a critical one.

Reply:
Ok, fantastic.